src/modules/aws_vpc/mappers/nat_gateway.ts

```diff
@@ -197,8 +197,9 @@ export class NatGatewayMapper extends MapperBase<NatGateway> {
           if (!input.AllocationId) throw new Error('Elastic ip need to be created first');
         } else if (!e.elasticIp && e.connectivityType === ConnectivityType.PUBLIC) {
           const elasticIp = new ElasticIp();
-          // Attach the same tags in case we want to associate them visualy through the AWS Console
+          // Attach the same tags in case we want to associate them visually through the AWS Console
           elasticIp.tags = e.tags;
+          elasticIp.region = e.region;
```

Comment:
that is related to the PR?

Reply:
Yes, the test for this mapper was consistently failing because it was trying to `db.update` with region `undefined`, that was inserting a `null` and the region cannot be `null` for the elastic ip entity